queries.sql

```diff
@@ -1,3 +1,17 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%pus';
+
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+
+SELECT name FROM animals WHERE neutered=true AND escape_attempts<2;
+
+SELECT date_of_birth FROM animals WHERE name='Saggapus' OR name='rex';
+
+SELECT name, escape_attempts FROM animals WHERE weight_kg>8;
+
+SELECT * FROM animals WHERE neutered=true;
+
+SELECT * FROM animals WHERE NOT name='Milu';
+
+SELECT * FROM animals WHERE weight_kg BETWEEN 10 AND 20;
```

Comment:
- [ ] Let's make sure you are following the [guidelines](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_create_tables.md) properly. 
<img width="885" alt="Capture" src="https://user-images.githubusercontent.com/58504655/135107343-01157dd4-87f9-4fb2-a93d-167ff9f74022.PNG">